lib/debug/chrome.dart

```diff
@@ -728,13 +719,13 @@ class PropertyDescriptor extends InternalPropertyDescriptor {
   String toString([String indent = '  ']) =>
       "PD: $name ${value?.toString(indent + '  ')}\n"
       "${indent}fg: w: $writable c:$configurable e:$enumerable t:$wasThrown o:$isOwn\n"
-      "${indent}get: $getFunction\n"
-      "${indent}set: $setFunction\n"
+      "${indent}get: ${getFunction.toString(indent + '  ')}\n"
```

Comment:
Hmm, not necessarily part of this PR, but it's odd to see `toString()` with an additional param. This is to map 1:1 to the with the JS chrome library?

Reply:
It's a trick I use to allow formatting toString output but since it covers the regular toString() then you can use it in regular print or string formatting.